content/updates/2020-10/3-Better-manage-product-models-and-variants-in-the-grid.md

```diff
@@ -1,26 +1,46 @@
 ---
-pim_announcement_alt_img: ""
+pim_announcement_alt_img: "TODO"
 pim_announcement_audience:
 - CE
 - EE
 ---
 
 # Better manage product models and variants in the grid
 
-TODO
+You work a lot with product models and their variants? You will enjoy these simple yet efficient new improvements we delivered in the product grid.
+
 ## Filter on the product typology
-::: meta-data type="New" features="Productivity" available="early October" link-to-doc="../articles/.html"
+::: meta-data type="New" features="Productivity" available="early October" link-to-doc="../articles/enrich-products-variants.html#filter-on-product-typology"
 :::
 
+The new `Product typology` filter in the product grid allows you to choose whether to display only the simple products or only the variant products. Pretty useful when you want to quickly find different variant products in different product models, for instance.
+
+![](../img/TODO.png)
+
 ## Product variant display option
-::: meta-data type="New" features="Productivity" available="early October" link-to-doc="../articles/.html"
+::: meta-data type="New" features="Productivity" available="early October" link-to-doc="../articles/TODO.html"
 :::
 
+The product grid also benefits from a new display option: the `Variant` display option. It allows you to show your models' product variants, instead of only these product models. This way, you can easily find and work at your product variant level. What a time saver!
+
+![](../img/TODO.png)
+
 ## Filter on the product identifier
-::: meta-data type="New" features="Productivity" available="early October" link-to-doc="../articles/.html"
+::: meta-data type="New" features="Productivity" available="early October" link-to-doc="../articles/TODO.html"
 :::
 
+Filtering on your product model identifiers has now arrived in your product grid! So neat!
```

Comment:
Only PM Id.

Reply:
I'm not sure it concerns the search bar. @morganen Does is work in the search bar too?

Reply:
So it applies to the system filter "Identifier" and in the search bar (I could filter on a PM ID through the search bar)?

Reply:
Not only PM: you can search on the sub PM identifier as well. And yes, it works also with the search bar :)

Reply:
Ok, so it's both thanks to the search bar and the identifier filter. In this case, I would change the title. And I think we need to change your paragraph, Laure, so that we can mention it.
```suggestion
You can now filter on your product model (and sub-product model) identifiers in your product grid! Enter your product model ID in our search bar or in the `Identifier` filter et voilà!
```
@LaureBro what do you think?

Reply:
Update : 
- le filtre permet de chercher sur l'identifier d'un product model ou d'un sous-produit model, ou d'un produit (simple ou variant. 
- dans la search bar, on peut chercher sur l'identifiant (= code) ou sur le label d'un product model, d'un sous-produit model et d'un produit (variant ou non). 
La seule nouveauté réside dans le fait que le nouveau filtre remonte aussi les parents.